pkg/object/httpserver/mux.go

```diff
@@ -44,6 +49,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	defaultAccessLogFormat = "[$Time] [$RemoteAddr $RealIP $Method $URI $Proto $StatusCode] [$Duration rx:$ReqSize tx:$RespSize] [$Tags]"
```

Comment:
for the log format, the current design, only support adding a prefix to a variable but does not support adding a suffix to a variable.
I propose we use `{{` and `}}` to mark a variable, that's, the default format becomes:

```suggestion
	defaultAccessLogFormat = "[{{Time}}] [{{RemoteAddr}} {{RealIP}} {{Method}} {{URI}} {{Proto}} {{StatusCode}}] [{{Duration}} rx:{{ReqSize}}B tx:{{RespSize}}] [{{Tags}}]"
```
note the `B` for `ReqSize` and `RespSize` are now part of the format.

we will need to update `newAccessLogFormatter` accordingly.